### DISCUSSION TITLE
Sonoma-Ventura-Monterey-OC-0.9.0-Stable_Release-Infinix-InBook-Series

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#37] [Discuss about the release]
This may be the last update even I am working for MacOS 15. But if anyone contributed for this project, I will update EFI for every release of macos. A small cotribution will also be appreciated. If anyone are ready to contribute their EFI here will be appriciated too. Because this repository is a main branch for our Infinix INBOOKS Hackintosh. So anyone can work in this repository about our all INFINIX INBOOKS.

- Updated all the required kexts.

- Generate Your SMBIOS if you need iservices like iMessages, etc.

- Fixed Bluetooth. Now Bluetooth is working as normal.

- Added battery Indication and detection like ventura. 

- Added HORNDIS supported kext, so you can connect your android to macbook and use android as ethernet. but you need to install [HORNDIs](https://github.com/jwise/HoRNDIS/releases/tag/rel9.2) in your mac.

- Fixed trackpad which is not working directly.

- Replaced back to AirportItlwm from Heliport. Because there is AirportItlwm kext for Sonoma. So no need to install Heliport.app.

- Everything is same as Previous EFI.

- Added all the required kexts.

- if you want this EFI to work on bigsur, then you need to delete all bluetooth kexts & wifi kexts then update config file. thats it.

**NOTE**: Donot update any kext. Keep it as it should be. Just change SMBIOS and Book picker. DON'T touch anything more than those two. Other wise it may make boot loop.

If you enjoyed this project — or just feeling generous, consider buying me a coffee. Cheers! ☕ :
- **If you enjoyed this project — or just feeling generous, consider buying me a coffee. Cheers! ☕ :**
# :diamond_shape_with_a_dot_inside:buy me a coffee :coffee:  Donate :  :diamond_shape_with_a_dot_inside:
donate via paypal :
<a href="https://www.paypal.me/369639/"><img src="blue.svg" height="40"></a>  
donate via Paytm,phonepe,gpay:
| <a><img src="darkblue.svg" height="40"></a> |
|-------------------------------------------|
|  `9392656181` |

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/'>Sonoma-Ventura-Monterey-OC-0.9.0-Stable_Release-Infinix-InBook-Series</a>.</em>

**Comment by barrrliebert** [comment]
I just swapped out my old SSD(built in) for a new ssd WDC SN 730. Should I make any changes to the EFI?

**Reply by devboloji** [reply]
need more information to answer your question. You are using WDC SN 730 as primary? are you using single SSD in your laptop?

**Reply by barrrliebert** [reply]
Yes, I only have 1 SSD in my laptop and it's the WDC SN730 as the primary

**Reply by devboloji** [reply]
not a problem, you can use any SSD as your wish. you can use old efi without modification

**Reply by barrrliebert** [reply]
so i can use this efi to install hackintosh?

**Reply by devboloji** [reply]
yes without any doubt. it works.
